### DISCUSSION TITLE
Configure a static ip address for RDZsonde

### DISCUSSION THREAD

**Discussion by OE5REO** [dl9rdz/rdz_ttgo_sonde#365] [Developers corner]
Is it possible to configure a STATIC IP for RDZsonde? I could not find anything in the configuration settings. Any help appreciated ...

**Comment by hr-upa** [comment]
There is no option for that in the configuration settings.

You could do that by a simple patch in the source code and then compiling yourself.

For the WiFi client modes, you would need to add something like
WiFi.config(local_IP, gateway, subnet, primaryDNS, secondaryDNS)
just before the line WiFi.begin() (two different places for the two Wifi client modes)

For the WiFi AP mode, a static IP, configured in the source code, is used. This can be changed in the line
WiFi.softAPConfig(IPAddress (192, 168, 4, 1), IPAddress (0, 0, 0, 0), IPAddress (255, 255, 255, 0)

(all in RX_FSK.ino)

**Comment by Ernie34** [comment]
You could also just set a static IP address in your router. For example I
have mine set as static and I have it port forwarded as well so I can
access it from my phone anywhere or an outside computer.

**Comment by OE5REO** [comment]
I'm not an expert in these kind of things, but I will do my best to change the IP settings. Thanks for your help!
